pkg/kubectl/cmd/cmd.go

```diff
@@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 
 func checkErr(err error) {
 	if err != nil {
-		glog.FatalDepth(1, err.Error())
+		if errors.IsStatusError(err) {
```

Comment:
Wait.... glog.Fatal from here is really bad.  We shouldn't have been doing that in the first place...